test/cmdlineTests.sh

```diff
@@ -165,10 +166,10 @@ function test_solc_behaviour()
     if [[ "$(cat $stdout_path)" != "${stdout_expected}" ]]
     then
         printError "Incorrect output on stdout received. Expected:"
-        echo "${stdout_expected}"
+        echo -e "${stdout_expected}"
 
         printError "But got:"
-        cat $stdout_path
+        echo -e "$(cat $stdout_path)"
```

Comment:
This `-e` is used to pretty print solidity source code in `.json` output, right?

Just my humble opinion, maybe we can make it even better: when the file name matches `*.json`, use `echo -e "$(cat $stdout_path | jq .)"` to pretty print it. (if `jq` or other similar tools presents)

Reply:
Hm, didn't know jq before, thanks for the hint!

Yes exactly, the solidity source code is really hard to read otherwise. Actually I think we should even store the output in that way if it is json, since the diffs are otherwise not really useful. As far as I can see, jq does not support strings with raw newlines, or is there an option for that?

Reply:
> jq does not support strings with raw newlines

strings with raw newlines probably not conform to JSON standard?

Reply:
Indeed!